src/routes/api/logout.route.js

```diff
@@ -0,0 +1,10 @@
+import { Router } from 'express';
+
+import Logout from '../../controllers/logout';
+
+const router = Router();
+const { logout } = Logout;
+
+router.get('/v1/auth/logout', logout);
```

Comment:
I just noticed the logout route is a `GET` request. Is there a reason why this is not a POST request?.
Also, I see you named your route file `logout.route.js`, every route file in the codebase doesn't have the `**.route.js`. Please update your code to follow the existing pattern.

Reply:
on the passport docs it was a get request that's why i followed suit